support-bundle/support-bundle-edge.sh

```diff
@@ -237,6 +241,30 @@ function stylus-files() {
   mkdir -p $TMPDIR/run/immucore
   ls -lah /run/immucore/ > $TMPDIR/run/immucore/files 2>&1
   cp -prf /run/immucore/* $TMPDIR/run/immucore 2>&1
+
+# collect content from /opt/spectrocloud/bin-checksums/*
+  techo "Collecting content from /opt/spectrocloud/bin-checksums/*"
+  mkdir -p $TMPDIR/opt/spectrocloud/bin-checksums
+  for file in /opt/spectrocloud/bin-checksums/*; do
+    if [ -f "$file" ]; then
+      cp -p "$file" "$TMPDIR/opt/spectrocloud/bin-checksums" 2>&1
+    fi
+  done
+
+  #  check if sha256sum is installed
+  if ! command -v sha256sum >/dev/null 2>&1; then
+    techo "sha256sum command not found"
+    return
+  fi
```

Comment:
I wonder if we should also try `openssl dgst -sha256` command if `sha256sum` is not installed and then quit.